rust-toolchain

```diff
@@ -1 +1 @@
-nightly-2021-12-14
\ No newline at end of file
+stable
```

Comment:
`stable` is a moving target. The filcrypto team usually pins to a specific version. In this case I'd use `1.59.0`.